From 501e6f9954faa5ef645604cc1ad7f7ba6b2a8133 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Wed, 12 Mar 2025 15:23:55 -0400
Subject: [PATCH] Remove pytest-runner from setup_requires

The `pytest-runner` package has been deprecated upstream for some time,
and the project is now archived:
https://github.com/pytest-dev/pytest-runner/tree/v6.0.1?tab=readme-ov-file#deprecation-notice

This does not affect running the tests with `pytest`.
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 02141590a..651a1e877 100644
--- a/setup.py
+++ b/setup.py
@@ -135,6 +135,5 @@ using a computer and a display/projector.""",
             'flake8',
         ]
     },
-    setup_requires=['pytest-runner'],
     entry_points={'gui_scripts': ['openlp = openlp.__main__:start']}
 )
-- 
GitLab

